src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -611,7 +611,7 @@ type Itinerary {
   walkTime: Long
 }
 
-type Leg {
+type Leg implements Node {
```

Comment:
I'm pretty sure this is backwards-compatible but lets discuss in the dev meeting.

Reply:
I'm almost 100% sure it is.

Reply:
I tried to do some research on this, but could not find anything. So, lets change it and see what happens - maybe wait with merging this PR after the next release?